azure-servicebus/setup.py

```diff
@@ -38,36 +35,54 @@
 except ImportError:
     pass
 
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, '__init__.py'), 'r') as fd:
+    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
+                        fd.read(), re.MULTILINE).group(1)
+
+if not version:
+    raise RuntimeError('Cannot find version information')
+
+with open('README.rst', encoding='utf-8') as f:
+    readme = f.read()
+with open('HISTORY.rst', encoding='utf-8') as f:
+    history = f.read()
+
 setup(
-    name='azure-servicebus',
-    version='0.21.1',
-    description='Microsoft Azure Service Bus Client Library for Python',
-    long_description=open('README.rst', 'r').read(),
-    license='Apache License 2.0',
+    name=PACKAGE_NAME,
+    version=version,
+    description='Microsoft Azure {} Client Library for Python'.format(PACKAGE_PPRINT_NAME),
+    long_description=readme + '\n\n' + history,
+    license='MIT License',
     author='Microsoft Corporation',
-    author_email='ptvshelp@microsoft.com',
+    author_email='azpysdkhelp@microsoft.com',
```

Comment:
Do you have a speficic email? For SB, or for your team? I'm fine using this one, as long as you are in there :)

Reply:
I don't think we have a separate email yet - but yes, I am on that DG :)